test/resources/dummy-activitystreams.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html lang=en-us>
+  <head>
+    <meta charset="utf8">
+  </head>
+  <body>
+      <p>Activity Streams!</p>
```

Comment:
Nit: whitespace. Any reason this is using so much? It then switches back to 2 spaces in the js.